challenges/objects-arrays.js

```diff
@@ -97,23 +144,25 @@ The zoos need a list of all their animal's names (names only, not scientific) co
 
 */
 
-const lowerCase = [];
+const lowerCase = zooAnimals.map( animal => animal.animal_name.toLowerCase() ); 
 console.log(lowerCase); 
 
 /* Request 3: .filter() 
 
 The zoos are concenred about animals with a lower population count. Find out which animals have a population less than 5.
 
 */
-const lowerPopulation = [];
+const lowerPopulation = zooAnimals.filter( animal => animal.population < 5 );
 console.log(lowerPopulation);
 
 /* Request 4: .reduce() 
 
 The zoos need to know their total animal population across the United States.  Find the total population from all the zoos using the .reduce() method.
 
 */
-const populationTotal = 0;
+const populationTotal = zooAnimals.reduce( (acc, current) => {
```

Comment:
Great work using advanced array methods and arrow syntax for this entire file. This last one can be written a little simpler and more semantic like this:

```
const populationTotal = zooAnimals.reduce( (total, animal) => total+ animal.population, 0);
```